homeassistant/components/tailwind/cover.py

```diff
@@ -77,6 +78,14 @@ async def async_open_cover(self, **kwargs: Any) -> None:
                 translation_domain=DOMAIN,
                 translation_key="door_locked_out",
             ) from exc
+        except TailwindDoorOperationError:
```

Comment:
I think this error isn't specific to just the current state of the door already in the requested state. I think it will throw the same error for other occurrences as well; for example, when the requested state change fails (like jams). With this change, those cases are now suppressed incorrectly.

Reply:
Indeed.  I created an [issue](https://github.com/frenck/python-gotailwind/issues/297) and will create a PR for gotailwind to raise a specific exception that is backward compatible..